Firestore/Source/API/FSTUserDataConverter.mm

```diff
@@ -557,11 +590,10 @@ - (nullable FSTFieldValue *)parseScalarValue:(nullable id)input context:(FSTPars
     if (*reference.databaseID != *self.databaseID) {
       const DatabaseId *other = reference.databaseID;
       FSTThrowInvalidArgument(
-          @"Document Reference is for database %@/%@ but should be for database %@/%@%@",
-          util::WrapNSStringNoCopy(other->project_id()),
-          util::WrapNSStringNoCopy(other->database_id()),
-          util::WrapNSStringNoCopy(self.databaseID->project_id()),
-          util::WrapNSStringNoCopy(self.databaseID->database_id()), [context fieldDescription]);
+          @"Document Reference is for database %s/%s but should be for database %s/%s%@",
```

Comment:
Thank you for making this kind of change: keeping the amount of portability marshaling/wrapping code to a minimum helps both for clarity and our eventual goal of migrating away from Objective-C

Reply:
ack